### PR TITLE
Eclipse Collections 11.0 release prep 

### PR DESCRIPTION
**Pull request by prathasirisha**
Signed-off-by: Sirisha Pratha <sirisha.pratha@bnymellon.com>

### CODE REVIEW

File: RELEASE_NOTE_DRAFT.md [hunk 1]
```diff
@@ -1,25 +1,112 @@
 11.0.0
 ====================
 
-This is the 11.0.0 major release. 
+This is the 11.0.0 major release.
+
+Central theme of this release is addition of new APIs, features as requested by the community and decrease Tech Debt.
+This is our yearly major release.
```
Comment (donraab) [inline RELEASE_NOTE_DRAFT.md:7]: The 10.0.0 release was in July, 2019. So this is the first major release in over two years.

**Review by donraab — CHANGES_REQUESTED** [review]
Just one note on our frequency of major releases. Otherwise LGTM!

File: RELEASE_NOTE_DRAFT.md (hunk 1 above)
Comment (prathasirisha) [reply RELEASE_NOTE_DRAFT.md:7]: Removed it.

File: RELEASE_NOTE_DRAFT.md [hunk 2]
```diff
@@ -1,25 +1,102 @@
...
+* Deleted version.properties as the file is no longer needed.
+* Removed extra calls to map.get in sumByDoubleFunction and sumByFloatFunction.
+* Removed implementations of OrderedIterable.toStack() that can use the default implementation instead.
+* Removed implementations of aggregateBy and aggregateInPlaceBy that can use default implementations instead.
+* Removed references to deleted classes in findbugs-exclude file.
+
+# Removed Functionality
+--------------------------
+* Removed Verify.assertThrows() which takes a Runnable in favor of Assert.assertThrows(). This is a breaking change.
+* Removed assertNotEquals() from Verify as Assert already has the same API.
```
Comment (nikhilnanivadekar) [inline RELEASE_NOTE_DRAFT.md:68]: `This is a breaking change`

File: RELEASE_NOTE_DRAFT.md [hunk 3]
```diff
@@ -1,25 +1,102 @@
...
+* Enabled Dependabot v2.
+* Added security badge in README.md.
+* Added GitHub action to generate code coverage report.
+* Added JDK Early Access GitHub Actions.
+* Added badges pointing to javadoc.io.
+* Added badges to website.
+
+# Breaking Changes
+-------------------------
+Warning: These changes are already mentioned above. The list below might not be exhaustive, make sure to test your application and usages to verify.
```
Comment (nikhilnanivadekar) [inline RELEASE_NOTE_DRAFT.md:94]: We should also have a line at the corresponding change saying Breaking Change.

**Review by nikhilnanivadekar — CHANGES_REQUESTED** [review]
Almost there, just a couple of changes. Fix them and feel free to merge it right before the release. Since it has Readme changes, you will want to merge it as close to the release as possible.

File: RELEASE_NOTE_DRAFT.md (hunk 2 above)
Comment (prathasirisha) [reply RELEASE_NOTE_DRAFT.md]: Done.

File: RELEASE_NOTE_DRAFT.md (hunk 3 above)
Comment (prathasirisha) [reply RELEASE_NOTE_DRAFT.md:95]: Done.

**Review by nikhilnanivadekar — APPROVED** [review]

**Pull request merged**